docbrown/src/db/graph_window.rs

```diff
@@ -798,6 +801,71 @@ impl<G: GraphViewOps> GraphViewInternalOps for WindowedGraph<G> {
         )
     }
 
+    fn temporal_vertex_timestamps_vec(
+        &self,
+        v: VertexRef,
+    ) -> Vec<i64> {
+        self.graph.temporal_vertex_timestamps_vec(
```

Comment:
I think the normal functions on the graph_window also need to call the `graph.temporal_vertex_timestamps_vec_window` using the times set within the window. Same with the edges below. I can go through this tomorrow with you if you like,